apps/allocations/app/store/allocation.js

```diff
@@ -73,24 +73,28 @@ const getAllocation = async ({ accountId, payoutId }) => {
   )
     .pipe(
       first(),
-      map(({
+      map(([{
+        budget,
+        token
+      },{
         amount,
-        description,
         distSet,
         period,
         recurrences,
-        startTime,
-        token,
-      }) => ({
+        startTime
+      },
+      description
+      ]) => ({
```

Comment:
😖 

This is _really_ hard to understand. I think some indentation cleanup might be all it takes to clarify.